content/assignments/hello-world/_index.md

```diff
@@ -0,0 +1,151 @@
+
+1. Inside `HelloWorldAutoGraded`, open up the `SayHelloClass.cs` class.  This is where you will write any code for this assignment.  Within this     class, you will see the following:
+   * The `SayHello()` method which is returning `&&&`.
+   * `SayHelloClass()` constructor which is empty.  This will remain empty for this assignment.
+
+
+1. Now, turn your attention to Program.cs in the `HelloWorldAutograded` project. This class will print two strings to the console using the `Console.WriteLine()` method.  
+   * The first returns the string value of the `SayHello()` method from the `SayHelloClass`.
+   * The second returns the data type of the `SayHello()` method’s output.
+
+
+1. You are now ready to run the program.  Do the following:
+   * **Mac Users:** Hit the Run button at the top right-hand corner of the window.  Right-clicking on the `HelloWorldAutograded` project and selecting Run Project will also work.
+   * **Windows Users:** Click on the solid green triangle next to the name of the `HelloWorldAutograded` project. You can also press F5 to run the project.
+
+
+1. Visual Studio opens a pane and displays any program output when you run a program. Currently, the program output contains the following:
+   
+   ``` console 
+   &&&
+   System.String
+   ```
+1. You are ready to run the auto-grading tests.
+
+## Running the Auto-grading Tests
+
+The tests provide feedback about your code before you submit your work.  Use the tests to check your work as you complete the assignment.
+
+The tests are unit tests for you to use to check your work.  We call them auto-graded tests instead of unit tests in these projects. 
+
+Let’s learn how to run the tests in Visual Studio.
+
+1. Each assignment will contain a test **project**.  This project contains all of the auto-grading tests and may have more than one test class.  In the Solution Explorer, navigate to HelloTests.cs inside the `Hello.Tests` project. 
+
+   {{% notice blue "Note" "rocket" %}}
+   You will not need to open the `Using.cs` file.  This file connects the testing project to the testing library.  This is not a test class.
+   {{% /notice %}}
+
+   A **test** is a single method in a test file with `[TestMethod]` above it. 
+
+      ```csharp
+      [TestClass]
+      public class AutogradingTests
+      {
+         [TestMethod]
+         public void TestToDemonstrateWhatATestMethodLooksLike()
+         {
+            //test code here
+         }
+      }
+      ```
+
+      A test class may contain many tests. 
+
+
+      ```csharp
+      [TestClass]
+      public class AutogradingTests
+      {
+         [TestMethod]
+         public void TestToDemonstrateWhatATestMethodLooksLike()
+         {
+            //test code here
+         }
+
+         [TestMethod]
+         public void SecondTestToDemonstrateWhatATestMethodLooksLike()
+         {
+            //test code here
+         }
+      }
+   ```
+1. Run all of your tests using the Test Explorer window.
+   1. Mac Users: 
```

Comment:
This opens a Tests Pane on the right hand side the window where one can navigate and run test all together or individually.